app/src/main/res/layout/activity_event_list.xml

```diff
@@ -7,6 +7,17 @@
     android:layout_height="match_parent"
     android:background="@android:color/white">
 
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:id="@+id/tv_android_id"
```

Comment:
@tuancoltech Can we make the Android ID selectable, so that it can be copied?

This value will be used by distributors as proof that they [distributed](https://sponsors.elimu.ai/distributions/add) a device to a student, so I can imagine that there might be a need to copy this value from the app.